devTools/Dockerfile-php80-xdebug

```diff
@@ -0,0 +1,33 @@
+reFROM php:8.0-rc-cli-alpine
+
+RUN apk add --no-cache \
+        bash \
+        expect \
+        zip
+
+ARG XDEBUG_VERSION=3.0.0beta1
+
+RUN set -eux; \
+    apk add --no-cache --virtual .build-deps \
+        ${PHPIZE_DEPS} \
+        build-base \
+    ; \
+    mkdir -p /usr/src/php/ext/xdebug && curl -fsSL https://pecl.php.net/get/xdebug-${XDEBUG_VERSION} | tar -xvz -C "/usr/src/php/ext/xdebug" --strip 1; \
```

Comment:
since php8 `pecl` and `pear` are not available anymore: https://externals.io/message/103977

Reply:
Yeah, that's one reason of my concerns.

Reply:
there is an option to use https://packagist.org/packages/friendsofphp/pickle instead but I do not think that this is good idea.

Reply:
That's correct.